browser/resources/settings/brave_overrides/basic_page.ts

```diff
@@ -308,6 +309,20 @@ RegisterPolymerTemplateModifications({
           prefs: '{{prefs}}'
         }
       ))
+      const sectionLeoCustomModels = document.createElement('template')
+      sectionLeoCustomModels.setAttribute('is', 'dom-if')
+      sectionLeoCustomModels.setAttribute('restamp', 'true')
+      sectionLeoCustomModels
+        .setAttribute('if', '[[showPage_(pageVisibility.leoCustomModels)]]')
```

Comment:
Where is `leoCustomModels` even defined? I don't see it in page_visibility.ts

Reply:
no where. it was a copy paste and i have no idea how this is working either.